broker/tasks/route53.py

```diff
@@ -414,3 +374,77 @@ def _delete_health_check(health_check_id):
             "Associated health check not found",
             extra={"health_check_id": health_check_id},
         )
+
+
+def _delete_ALIAS_record(domain, service_instance):
+    alias_record = f"{domain}.{config.DNS_ROOT_DOMAIN}"
+    target = service_instance.domain_internal
+    logger.info(f'Removing ALIAS record {alias_record} pointing to "{target}"')
+
+    try:
+        route53_response = route53.change_resource_record_sets(
+            ChangeBatch={
+                "Changes": [
+                    {
+                        "Action": "DELETE",
+                        "ResourceRecordSet": {
+                            "Type": "A",
+                            "Name": alias_record,
+                            "AliasTarget": {
+                                "DNSName": target,
+                                "HostedZoneId": service_instance.route53_alias_hosted_zone,
+                                "EvaluateTargetHealth": False,
+                            },
+                        },
+                    },
+                    {
+                        "Action": "DELETE",
+                        "ResourceRecordSet": {
+                            "Type": "AAAA",
+                            "Name": alias_record,
+                            "AliasTarget": {
+                                "DNSName": target,
+                                "HostedZoneId": service_instance.route53_alias_hosted_zone,
+                                "EvaluateTargetHealth": False,
+                            },
+                        },
+                    },
+                ]
+            },
+            HostedZoneId=config.ROUTE53_ZONE_ID,
+        )
+    except:  # noqa E722
```

Comment:
I left this error handling in, but it seems quite aggressive. It seems like we do want to know if we get an error because of missing IAM permissions or something similar

Reply:
I think the reasoning was that abandoned records are pretty harmless (same as why we don't track the change IDs and just assume Route53 will delete them successfully)

Reply:
The available exceptions for this request don't seem to make it very easy to detect different classes of errors

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/route53/client/change_resource_record_sets.html

Reply:
I guess overall, I'd like to be cleaning up Route53 vs leaving irrelevant records in there. As is, we have no way of knowing if that is working or if it is failing silently. So all of the changes in this PR could do nothing and we'd have no way of knowing